src/test/java/org/folio/rest/impl/MockServer.java

```diff
@@ -367,7 +373,8 @@ private void handleGetAcquisitionsMemberships(RoutingContext ctx) {
 
       AcquisitionsUnitMembershipCollection memberships;
       try {
-        memberships = new JsonObject(ApiTestBase.getMockData(ACQUISITIONS_MEMBERSHIPS_COLLECTION)).mapTo(AcquisitionsUnitMembershipCollection.class);
+        memberships = new JsonObject(ApiTestBase.getMockData(ACQUISITIONS_MEMBERSHIPS_COLLECTION)).mapTo(
+          AcquisitionsUnitMembershipCollection.class);
```

Comment:
Are you using the same code format as everyone else (max line length, etc.)?  Changes like this are just noise when reviewing PRs...